### PR TITLE
Remove backdrop when no onAction (#436)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
Addresses #436.

- added a classname for the label to allow for easy styling of label

### CODE REVIEW

File: src/taipy/gui/version.json [hunk 1]
```diff
@@ -1 +1 @@
-{"major": 2, "minor": 0, "patch": 0, "ext": "dev0"}
+{"major": 2, "minor": 1, "patch": 0, "ext": "dev0"}
```
Comment (dinhlongviolin1) [inline src/taipy/gui/version.json:1]: Thanks :D

**Review by dinhlongviolin1 — APPROVED** [review]
Well done!

File: gui/src/components/Taipy/Image.tsx [hunk 2]
```diff
@@ -147,18 +129,18 @@ const Image = (props: ImageProps) => {
...
                 className={className}
                 id={id}
             >
                 <ImageSrc style={imgStyle} />
-                <ImageBackdrop className="MuiImageBackdrop-root" />
+                {onAction ? <ImageBackdrop className="MuiImageBackdrop-root" /> : null}
                 {label === undefined ? null : (
                     <ImageSpan>
-                        <Typography component="span" variant="subtitle1" color="inherit" sx={imgSx}>
+                        <Typography component="span" variant="subtitle1" color="inherit" sx={imgSx} className={getSuffixedClassNames(className, "-label")}>
```
Comment (FabienLelaquais) [inline gui/src/components/Taipy/Image.tsx:143]: This -label thing should be documented somehere right?
Can you tell me about it?
Thx!

**Review by FabienLelaquais — APPROVED** [review]
👍

File: gui/src/components/Taipy/Image.tsx (hunk 2 above)
Comment (FredLL-Avaiga) [reply gui/src/components/Taipy/Image.tsx:143]: for ease of use, we append "-label" to each css class declared for the component (taipy-image + the user specified ones)

Comment (FredLL-Avaiga) [reply gui/src/components/Taipy/Image.tsx:143]: I guess it should be more widespread ...
It has been done in a few other places.

**Pull request merged**